irods_testing_environment/test_runner.py

```diff
@@ -227,9 +228,10 @@ def execute_test(self, test, reporter='junit'):
         output_dir = os.path.join(context.irods_home(), 'log')
         output_path = os.path.join(output_dir, f'{test}_{reporter}_report.{extension}')
 
-        cmd = [os.path.join(context.unit_tests(), test),
-               '--reporter', reporter,
-               '--out', output_path]
+        cmd = [os.path.join(context.unit_tests(), test)]
+
+        cmd.extend(options or ['--reporter', reporter, '--out', output_path])
```

Comment:
what does splitting this change?  anything?

if nothing chnaged... do you find it easier to read?